Exception in thread "main" java.lang.NumberFormatException: For input string: ""
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:678)
	at java.base/java.lang.Integer.parseInt(Integer.java:786)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:992)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
	at org.starloco.locos.quest.Quest.handleObjectives(Quest.java:73)
	at org.starloco.locos.quest.Quest.<init>(Quest.java:41)
	at org.starloco.locos.database.data.game.QuestData.lambda$loadFully$0(QuestData.java:22)
	at org.starloco.locos.database.data.FunctionDAO.getData(FunctionDAO.java:82)
	at org.starloco.locos.database.data.game.QuestData.loadFully(QuestData.java:18)
	at org.starloco.locos.game.world.World.createWorld(World.java:356)
	at org.starloco.locos.kernel.Main.start(Main.java:78)
	at org.starloco.locos.kernel.Main.main(Main.java:65)
